replibyte/src/source/mod.rs

```diff
@@ -10,7 +11,12 @@ pub mod postgres_stdin;
 pub trait Source: Connector {
     fn read<F: FnMut(OriginalQuery, Query)>(
         &self,
-        transformers: &Vec<Box<dyn Transformer + '_>>,
+        options: SourceOptions,
         query_callback: F,
     ) -> Result<(), Error>;
 }
+
+pub struct SourceOptions<'a> {
+    pub transformers: &'a Vec<Box<dyn Transformer>>,
+    pub skip_config: Option<Vec<SkipConfig>>,
```

Comment:
I would remove the `Option<Vec<..>>` and just keep the `Vec<...>` even if we have to create an empty vector

Reply:
👍